package.json

```diff
@@ -11,21 +11,34 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
-    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+    "test": "jest"
   },
   "dependencies": {
     "gatsby": "^2.23.3",
     "react": "^16.12.0",
     "react-dom": "^16.12.0"
   },
   "devDependencies": {
-    "prettier": "2.0.5"
+    "@babel/preset-env": "^7.10.3",
+    "@babel/preset-react": "^7.10.1",
+    "@testing-library/jest-dom": "^5.10.1",
+    "@testing-library/react": "^10.4.1",
+    "@testing-library/user-event": "^12.0.7",
+    "babel-jest": "^26.1.0",
+    "jest": "^26.1.0",
+    "jest-dom": "^4.0.0",
+    "nock": "^12.0.3",
+    "prettier": "2.0.5",
+    "react-test-renderer": "^16.13.1"
```

Comment:
I forget - is `react-test-renderer` required when testing with Jest/react-testing-library`?

Reply:
Oh, no it's not! I'm not sure why I have that it there because my suite does work without it